### PR TITLE
create docs : c program to take user input and toggle case

### PR DESCRIPTION
**Pull request by Dtech-Dbug**
- incl. headers and libraries besides ``` <stdio.h> ``` to facilitate user Input process
- converts lowercase to uppercase, manual algorithm.
- takes input from user via CLI

### CODE REVIEW

**Comment by dibashthapa** [comment]
`#include <cs50.h>`
@Dtech-Dbug 
Please remove this header file

**Comment by Dtech-Dbug** [comment]
@dibashthapa 
copy that :D

needs a few changes to the c program after this deletion.

File: C-Programs/greetAndToggleCase.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+const int ASCII_DIFFERENCE = 32;
+
+int main (void)
+{
+    int UPPERCASE;
+
+  string name = get_string("Input: ");
```
Comment (PrajwalCH) [inline C-Programs/greetAndToggleCase.c:11]: String doesn't exist in C. It's only available in cpp.

**Review by PrajwalCH — CHANGES_REQUESTED** [review]
use cpp if you want to use string. It's not valid C program.

**Comment by Dtech-Dbug** [comment]
> use cpp if you want to use string. It's not valid C program.

that was a helper from a header file - that i deleted

**Pull request closed without merging**